docs/examples/basic-with-apply.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Basic example, apply button
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This example is quite similar to the [basic example](basic), but instead of applying the changes real time, the user has to click on an 'apply' button to see the updates reflected.
```

Comment:
This example is quite similar to the [basic example](basic), but instead of applying the changes in real time, the user has to click on an 'apply' button to see the updates reflected.